src/databases/XYZ/avtXYZWriter.C

```diff
@@ -193,7 +195,7 @@ avtXYZWriter::WriteChunk(vtkDataSet *ds, int chunk)
         double *coord = ds->GetPoint((vtkIdType)a);
 
         // Get a viable atomic number
-        int atomicNumber = 0;
+        int atomicNumber = -1;
```

Comment:
I don't believe this has any impact on the eventual behavior, but I'm fine with the change.